drivers/include/net/netdev2/lorawan.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) Fundación Inria Chile 2017
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     drivers_netdev_netdev2
+ * @{ *
+ * @file
+ * @brief       Definitions for netdev2 common lorawan code
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ */
+
+#ifndef NETDEV2_ETH_H
+#define NETDEV2_ETH_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "net/netdev2.h"
+#include "net/netopt.h"
+#include "net/gnrc/lorawan/timer.h"
+#include "net/gnrc/lorawan/definitions.h"
+#include "net/gnrc/pkt.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define KEY_SIZE		16	
+
+#define LORAMAC_PHY_MAXPAYLOAD                      255
+#define LORA_MAC_COMMAND_MAX_LENGTH                 15
+
+#define GNRC_LORAWAN_JOIN_REQUEST_MSG 1001
+#define GNRC_LORAWAN_LINK_CHECK_MSG 1002
+
+/*!
+ * LoRaMAC channels parameters definition
+ */
+typedef union uDrRange
+{
+    int8_t Value;
+    struct sFields
+    {
+        int8_t Min : 4;
+        int8_t Max : 4;
+    }Fields;
+}DrRange_t;
```

Comment:
- space after ```}```

Are these definitions imported from somewhere?
If not, please don't use CamelCase, don't use type specifiers (*s*Fields), and please use lower case variable/field names.